.github/workflows/test-env-deploy.yml

```diff
@@ -171,7 +181,8 @@ jobs:
           echo ${{steps.get_tags.outputs.result}}
 
   testmo-report-preparation:
-    needs: prepare-tests
+    needs: [check-secret, prepare-tests]
+    if: needs.check-secret.outputs.testmo-url == 'true'
```

Comment:
Why not simply do this?

```
if: github.repository == 'saleor/saleor-dashboard'
```

Reply:
Will that do the same when it comes to community PRs?

Reply:
That's only for `push` event, I see it's a `pull_request` event, thus you could use:

```
github.repository == github.event.pull_request.head.repo.full_name
```

or

```
github.event.pull_request.head.repo.fork == false
```